keras_eval/utils.py

```diff
@@ -251,6 +251,15 @@ def show_results(results, concepts, id='default_model', mode='average', csv_path
 
 
 def ensemble_models(models, input_shape, combination_mode='average', ensemble_name='ensemble'):
+    '''
```

Comment:
I though we were using Google Docstring format as Python documentation format. Why switching now to a different format?

Reply:
New computer... I got the defaults. Good catch!